eos/rare-b-decays/bs-to-phi-ll.cc

```diff
@@ -1095,35 +1106,562 @@ The azimuthal angle between the Kbar-K plane and the l^+l^- plane.";
 
         return BsToPhiDileptonAndConjugate::AngularhCoefficients(integrated_angular_h_coefficients_array);
     }
+    BsToPhiDileptonAndConjugate::AngularhCoefficients
+    BsToPhiDileptonAndConjugate::differential_angular_h_coefficients(const double & s) const
+    {
+        return BsToPhiDileptonAndConjugate::AngularhCoefficients(differential_angular_h_coefficients_array(s));
+    }
 
     double
-    BsToPhiDileptonAndConjugate::integrated_H_1c(const double & s_min, const double & s_max) const
+    BsToPhiDileptonAndConjugate::integrated_H(const double & s_min, const double & s_max, const std::string & name) const
```

Comment:
We need to discuss this in more details :)

Reply:
In the most recent version i have one suggestion using an enum instead of a string to curry the function instead. (See the integrated and differential S functions).